### PR TITLE
Add recipe - Masala Chaas 

### PR DESCRIPTION
**Pull request by shaktibarath**

### CODE REVIEW

File: src/recipes/masala-chaas.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            "quantity": "1/4",
+            "measure": "tablespoon",
+            "ingredient": "chat masala"
+        }
+    ],
+    "directions": [
+        "Add all ingredients to a blender, then Blend till combined, and Pour into serving glasses, garnish with fresh cilantro or mint and serve.",
+        "Enjoy!"
+    ],
+    "image": "blackthorn-cocktail.jpg",
```
Comment (alfg) [inline src/recipes/masala-chaas.json:51]:
```suggestion
    "image": "masala-chaas.jpg",
```

I also can't seem to load the added image in this PR. Is it corrupted?

File: src/recipes/masala-chaas.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "ingredient": "black salt"
+        },
+        {
+            "quantity": "1/4",
+            "measure": "tablespoon",
+            "ingredient": "chat masala"
+        }
+    ],
+    "directions": [
+        "Add all ingredients to a blender, then Blend till combined, and Pour into serving glasses, garnish with fresh cilantro or mint and serve.",
```
Comment (alfg) [inline src/recipes/masala-chaas.json:48]: Can we separate `Garnish with fresh...` into another step?

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/masala-chaas.json (hunk 2 above)
Comment (shaktibarath) [reply src/recipes/masala-chaas.json]: made the changes

**Comment by alfg** [comment]
Can you replace the image? It seems to be corrupted.

**Comment by shaktibarath** [comment]
> Can you replace the image? It seems to be corrupted.

Uploaded a new image.

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**